src/tools/parseUsername.ts

```diff
@@ -0,0 +1,5 @@
+function parseUsername(nick: string): string {
```

Comment:
Any reason this can't just be where it's used directly? Assuming it's only used once

Reply:
The server (almost?) never sends usernames directly, but rather sends nicks for kicks/messages/logins, so I end up using it quite a lot of times.

Reply:
What about adding it to the User class?

Reply:
I think it makes the most sense to add it as a getter, but then again we would run into the problem where `getters` is any. Do you think it's worth it? I doubt we shouldn't use getters at all.
I think I agree that adding it to the User class is better than how it is right now.

Reply:
Just put it on the user for now - we can revisit after implementing class-based VueX

Reply:
I agree with putting it on the user class, but why did I want to put in getters?
The way I think of it now, getters are used to process the state and return values based on it, with parameters to the getter function only making sense as accessors, telling you which channel you want to get the info on for example.
So I don't think that a function which simply processes a string and returns another string would fit there, since it processes it in a purely functional way, based on a protocol, and not the state.

Reply:
Yeah, no, it shouldn't be a getter. I mean, getters shouldn't even _have_ arguments, right? (In the case you described, wouldn't you just do something like `.channels['channel'].someGetter`?